easybuild/easyconfigs/m/Multiwfn/Multiwfn-3.4.1-intel-2017b.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'MakeCp'
+
+name = 'Multiwfn'
+version = '3.4.1'
+
+homepage = 'http://sobereva.com/multiwfn/'
+description = "A multifunctional wavefunction analyzer"
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+source_urls = ['http://sobereva.com/multiwfn/misc/']
+sources = ['%(name)s_%(version)s_src_Linux.zip']
+checksums = ['07141333f59a4bb3e3855a0c6c2ac61ae0aa9009bd40b375848ea2b641d4ab8f']
+
+prebuildopts = '/bin/bash noGUI.sh && cd noGUI && '
```

Comment:
Is there a particular reason why you don't just use `./noGUI.sh` here? Do you *need* to run this in a subshell?

If the script is missing the right permissions, you could do `chmod u+x noGUI.sh && ./noGUI.sh`?

Reply:
The noGUI.sh script is executable but lacks a shebang. I just wanted to make sure it runs in bash.

Reply:
Maybe mention that in a comment above this line to clarify?